app/views/pages/core_materials_info.html.erb

```diff
@@ -0,0 +1,225 @@
+<% content_for :before_content, govuk_back_link(text: "Back", href: :back) %>
```

Comment:
Do we have a page title here?

Reply:
There is no page title in the content file. I'll think of some :)